### PR TITLE
fix: Fixing Rating focus border styling

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR is to add focus border styling to Rating.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a0bb58f2ac1d9f50bce33b7a9c4fb6b09922a956'>a0bb58f2ac1d9f50bce33b7a9c4fb6b09922a956</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-rating-preview/src/components/RatingItem/useRatingItemStyles.styles.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { RatingItemSlots, RatingItemState } from './RatingItem.types';
 import { tokens } from '@fluentui/react-theme';
+import { createFocusOutlineStyle } from '@fluentui/react-tabster';
```
Comment (sopranopillow) [inline packages/react-components/react-rating-preview/src/components/RatingItem/useRatingItemStyles.styles.ts:6]: nit: We generally try to keep type imports at the bottom.

**Review by sopranopillow — COMMENTED** [review]
Would you mind adding some screenshots of before and after? :)

**Comment by behowell** [comment]
> Would you mind adding some screenshots of before and after? :)

@sopranopillow Not exactly what you asked for, but in case you didn't know, you can see the docsite with changes in this PR if you click Details next to **Pull request demo site** in the list of checks.

**Review by behowell — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Pull request merged**